lib/perfdata/influxdbwriter.cpp

```diff
@@ -289,6 +289,7 @@ void InfluxdbWriter::SendPerfdata(const Dictionary::Ptr& tmpl, const Checkable::
 		fields->Set(String("acknowledgement"), FormatInteger(checkable->GetAcknowledgement()));
 		fields->Set(String("latency"), cr->CalculateLatency());
 		fields->Set(String("execution_time"), cr->CalculateExecutionTime());
+		fields->Set(String("unit"), pdv->GetUnit());
```

Comment:
That's not entirely correct here. Above is a loop over all performance data pairs, and this one just uses the last unit value.

Reply:
Ok, thx, i changed it to above loop.